app/assets/stylesheets/problems.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Problems controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
Empty file, pls dot not include empty files in repo, we will be kept here forever